mainapp/templates/mainapp/contributor_form.html

```diff
@@ -40,9 +37,161 @@
   </button>
   {% endbuttons %}
   </form>
-<script>
+ <h4>More Donation Options:</h4>
+   <table>
+    <thead>
+      <th style="width: 10%;"></th>
+      <th style="width: 30%;">PAYMENT GATEWAY SERVICES</th>
+      <th style="width: 30%;">ACCOUNT DETAILS</th>
+      <th style="width: 30%;">UPI QR CODE</th>
+    </thead>
+    <tbody>
+      <tr>
+        <td>
+          <img class="payment-logo" src="https://donation.cmdrf.kerala.gov.in//assets/images/gateway-southindianbank.jpg"/>
+          <br/>
+          <b>SOUTH INDIAN BANK</b>
+        </td>
+        <td>
+            Internet Banking - 50 banks supported<br/>
+            Domestic Credit and Debit Cards supported<br/><br/>
+
+            payment charges: Nil
+        </td>
+        <td>
+            A/c Number : 0721053000002584<br/>
+            A/c Name: Chief Minister’s Distress Relief Fund<br/>
+            Branch : Corporate Branch, Thiruvananthapuram<br/>
+            IFSC : SIBL0000721
+        </td>
+        <td>
+            <img class="qr-code" src="https://donation.cmdrf.kerala.gov.in//assets/images/upi-southindianbank.png"/>
+            <br/><br/>
+            keralacmdrf@sib
+        </td>
+      </tr>
+       <tr style="background-color: #ff5252">
+          <td>
+            <img class="payment-logo" src="https://donation.cmdrf.kerala.gov.in//assets/images/gateway-federalbank.jpg"/>
+            <br/>
+            <b>FEDERAL BANK</b>
+          </td>
+          <td>
+              <strong>TEMPORARILY OUT OF ORDER DUE TO FLOODS</strong><br /><br />
+              Internet Banking - 53 banks supported<br/>
+              Domestic Credit and Debit Cards supported<br/>
+              UPI payment supported<br/><br/>
+               payment charges: Nil
+          </td>
+          <td>
+              A/c Number : 10210100412397<br/>
+              A/c Name: Chief Minister’s Distress Relief Fund<br/>
+              Branch: Palayam, Thiruvananthapuram<br/>
+              IFSC : FDRL0001021
+          </td>
+          <td>
+              <img class="qr-code" src="https://donation.cmdrf.kerala.gov.in//assets/images/upi-federalbank.png"/>
+              <br/><br/>
+              cmdrf@fbl
+          </td>
+        </tr>
+         <tr>
+            <td>
+              <img class="payment-logo" src="https://donation.cmdrf.kerala.gov.in//assets/images/gateway-airtel.jpg"/>
+              <br/>
+              <b>AIRTEL PAYMENTS BANK</b>
+            </td>
+            <td>
+                Airtel Payments bank<br/>
+                Airtel Wallet<br/><br/>
+                 payment charges: Nil<br/>
+            </td>
+            <td>
+            </td>
+            <td>
+                <img class="qr-code" src="https://donation.cmdrf.kerala.gov.in//assets/images/upi-airtel.png"/>
```

Comment:
There are double slashes here (and other assets links). They don't cause any problems, but just FYI.

Reply:
Oh. Ok. My bad. Should I rectify that now? Or is it fine if I correct it the next time I edit something?

Reply:
Later is fine

Reply:
Would be better to host the images locally, adds unnecessary dependency on the CMDRF site, it was down when I was checking, causing broken images

Reply:
@jebu Waiting for @arjunkomath to upload images to S3

Reply:
@aravindharinair Its done, I've added the links.